inference-engine/src/cldnn_engine/cldnn_program.cpp

```diff
@@ -92,7 +92,7 @@ bool Program::CanProcessDynBatch(std::vector<std::shared_ptr<ngraph::Node>> ops,
     return true;
 }
 
-Program::Program(InferenceEngine::CNNNetwork& network, std::shared_ptr<const cldnn::engine> engine, const Config& config)
+Program::Program(InferenceEngine::CNNNetwork& network, std::shared_ptr<const cldnn::engine> engine, const Config& config, bool createTopologyOnly)
```

Comment:
Yes we tried that.. however there were a lot of dependencies to extract topology build out, i.e., it was adding topology nodes to program, so that part should be separated. I think it is better to handle it in separate work.

Reply:
For now, I added TODO comment for later fix.